pontoon/base/static/js/locale_manage.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
+
+  // Before submitting the form, update translators and managers
+  $('form').submit(function (e) {
+    $.each(['translators', 'managers'], function(i, value) {
```

Comment:
What's our browser support? `Array.forEach` has been around for a long time and is supported in modern browsers.